test/dlc_tests/sanity/test_pre_release.py

```diff
@@ -197,6 +208,71 @@ def test_emacs(image):
     _run_cmd_on_container(container_name, ctx, "emacs -version")
 
 
+@pytest.mark.model("N/A")
+def test_cuda_paths(gpu):
+    """
+    Test to ensure directory structure for GPU files has cuda version in it
+
+    :param gpu: gpu image uris
+    """
+    image = gpu
+    if "example" in image:
+        pytest.skip("Skipping example containers which are not explicitly tied to a cuda version")
+
+    dlc_path = os.getcwd().split('/test/')[0]
+    job_type = "training" if "training" in image else "inference"
```

Comment:
Please run `black` or something to standardize the entire file to PEP8 standards.

This particular comment is about the use of `'<str>'` vs `"<str>"`.

Reply:
sure, will do